README.md

```diff
@@ -14,11 +14,11 @@ This extension integrates [include-what-you-use](https://github.com/include-what
 
 ![IWYU](https://helly25.com/wp-content/uploads/2023/05/iwyu-animated.gif)
 
-2) Manually optimize the include files for the current C/C++ file by pressing `cmd-shit-P` + `I` + `W` + `Y` + `U` for command "`Include What You Use (current file)`". The extension will then lookup the current editor's file in the compile_commands.json file. If found it will then run the include-what-you-use tool. If that is successful it will then call the fix_includes.py script to apply the changes.
+2) Manually optimize the include files for the current C/C++ file by pressing `cmd-shift-P` + `I` + `W` + `Y` + `U` for command "`Include What You Use (current file)`". The extension will then lookup the current editor's file in the compile_commands.json file. If found it will then run the include-what-you-use tool. If that is successful it will then call the fix_includes.py script to apply the changes.
```

Comment:
I had a hard time fixing this one because I thought it was hilarious 😆

Reply:
:-)